airflow/jobs/triggerer_job.py

```diff
@@ -245,9 +245,11 @@ async def arun(self):
             await self.cleanup_finished_triggers()
             # Sleep for a bit
             await asyncio.sleep(1)
-            # Every minute, log status
+            # Every minute, log status if at least one trigger is running.
             if time.time() - last_status >= 60:
-                self.log.info("%i triggers currently running", len(self.triggers))
+                count = len(self.triggers)
+                if count > 0:
+                    self.log.info("%i triggers currently running", count)
```

Comment:
+1 to this change, Airflow logs are quite full of negative logging which border on spam IMHO. I've recently moved some to debug and got rid of others altogether (depending on the usefulness of the message in question).
 
Note: could this not just be simplified to:
```suggestion
                if self.triggers:
                    self.log.info("%i triggers currently running", count)
```

Reply:
But then you don't get an assignment to `count` –  !

Reply:
Ah yes of course, brain-o on my part! (although you could still save the line by taking the len inside the call to `log.info` :smile: )